bigtable/client/mutations.h

```diff
@@ -62,29 +62,140 @@ constexpr std::int64_t ServerSetTimestamp() { return -1; }
  * @name Create mutations to delete a range of cells from a column.
  *
  * The following functions create a mutation that deletes all the
- * cells in the given column family and column within the given
- * timestamp range.
+ * cells in the given column family and, column within the given
+ * timestamp in the range.
+ *
+ * The function accepts any instantiation of `std::chrono::duration<>` for the
+ * @p timestamp_begin and @p timestamp_end parameters.  For example:
+ *
+ * @code
+ * using namespace std::chrono_literals; // C++14
+ * bigtable::DeleteFromColumn("fam", "col", 0us, 10us)
+ * @endcode
  *
  * The ending timestamp is exclusive, while the beginning timestamp is
- * inclusive.  That is, the interval is [@p timestamp_begin, @p timestamp_end).
+ * inclusive.  That is, the interval is [@p timestamp_begin, @p timestamp_end].
```

Comment:
Is the timestamp range inclusive of the endpoint?  If so, then the typical interval notation is `[a, b)` as opposed to `[a, b]`, right?

https://www.basic-mathematics.com/interval-notation.html